shared/chat/inbox/row/small-team/index.js

```diff
@@ -39,23 +39,48 @@ type Props = {
   youNeedToRekey: boolean,
 }
 
+type State = {
+  isHovered: boolean,
+}
+
 const SmallTeamBox = isMobile
-  ? Box
+  ? ClickableBox
   : glamorous(Box)({
       '& .small-team-gear': {display: 'none'},
       ':hover .small-team-gear': {display: 'unset'},
       ':hover .small-team-timestamp': {display: 'none'},
     })
 
-class SmallTeam extends React.PureComponent<Props> {
+class SmallTeam extends React.PureComponent<Props, State> {
+  state = {
+    isHovered: false,
```

Comment:
why can't we just use css for this?

Reply:
Is there a way to do it with CSS and it override `props.backgroundColor`? That's the wall I ran into.

Reply:
instead of inlining the color you could just have it be in the css and use a classname, so the hover / not-hover would be defined using a classname so like the component would just have 
<SmallTeamBox className={this.props.selected ? 'selected-row' : 'row' />. this is likely a larger change

Reply:
The other part of the problem is that we need to plumb the color through to `Avatars` where it uses the background color for the border on multi-avatars. So, I figured I'd do it in one place and use it on both.

Reply:
@chrisnojima, do you have any suggestions (WRT my last comment)?